src/librustc_back/target/mips_unknown_linux_musl.rs

```diff
@@ -27,6 +27,7 @@ pub fn target() -> TargetResult {
             features: "+mips32r2,+soft-float".to_string(),
             max_atomic_width: Some(32),
 
+            crt_static_default: false,
```

Comment:
Note that because mips musl does not inherit from `linux_musl_base` (it goes straight to `linux_base`), I'm pretty sure this override isn't necessary since it picks up the global default (which is `false`).

Which also means that if you do want anything from `linux_musl_base` (e.g. `-Wl,--eh-frame-hdr`), you need to pull it in directly.